recipes/ncbi_taxid/run_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export LC_ALL=en_US.utf-8
+export LANG=en_US.utf-8
+
+diamond_add_taxonomy --help |grep DIAMOND_OUTPUT_FILE >/dev/null 2>&1
```

Comment:
If this is the binary name and the name of the repo, shouldn't the package name match? Just to make it more intuitive to find things?

Reply:
Yep - I'm not great with names and while the code can do more than deal with DIAMOND, right now that is its primary purpose so I changed everything to be `diamond_add_taxonomy`.